samples/excel/tables/apply-values-filter.yaml

```diff
@@ -0,0 +1,137 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Apply a values filter
+description: Apply a values filter to a table
+script:
+  content: |
+    $('#create-table').click(createTable);
+    $('#filter-data').click(filterData);
+
+    /** Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to add a new worksheet
+                sheet = context.workbook.worksheets.add();
+
+                // Call the helper to create the table
+                await queueCreateTable(sheet);
+
+                //Queue a command to activate the sheet
+                sheet.activate();
+
+                // Run all of the queued up commands
+                await context.sync();
+            }
+            catch (error) {
+                OfficeHelpers.Utilities.log(error);
+            }
+        })
+    }
+
+
+    /** Filter the data in the table*/
+    function filterData() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to get the sheet
+                sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                // Call the helper to filter the table
+                await queueFilterData(sheet);
+
+                // Run all of the queued up commands
+                await context.sync();
+            }
+            catch (error) {
+                OfficeHelpers.Utilities.log(error);
+            }
+        })
+    }
+
+    // Create the table
+    async function queueCreateTable(sheet: Excel.Worksheet) {
```

Comment:
Include in setup function with resetWorksheet helper as inhttps://github.com/WrathOfZombies/samples/blob/master/samples/excel/range/set-get-values.yaml